### PR TITLE
Fix: Added archived module

### PR DESCRIPTION
**Pull request by hmanwani-rh**
Context in Slack thread: https://redhat-internal.slack.com/archives/C04CUSD4JSG/p1726856912238529

The added module was archived in PR: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/389

**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):**
1.2, 1.3

**Issue:**

**Link to docs preview:**
Find the link in the PR

**Reviews:**
- [x] SME: @jerolimov 
- [x] Docs review: @themr0c 

**Additional information:**

### CODE REVIEW

**Review by themr0c — APPROVED** [review]

**Review by jerolimov — APPROVED** [review]
I guess this section was removed, because the theme is mentioned in more details below.

But there everything was related to colors. So it's correct that we need to restore this section for the icons (`fullLogo` and `iconLogo`), while the theme is already described in "7.5. Customizing the theme mode color palettes for your Developer Hub instance
".

I'll add lgtm because its an improvement, also if **_I_** think we should remove everything related to colors from "7.2. Customizing the branding of your Developer Hub instance".

lgtm!

**Review by jerolimov — APPROVED** [review]
Merged this into one section lgtm!

/lgtm

File: modules/customization/proc-customize-rhdh-branding-logo.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// assembly-customize-rhdh-theme.adoc 
+//restored module as per slack thread: https://redhat-internal.slack.com/archives/C04CUSD4JSG/p1726856912238529
+
+[id="proc-customize-rhdh-branding-logo_{context}"]
+= Customizing the branding logo of your {product-short} instance
+
```
Comment (themr0c) [inline modules/customization/proc-customize-rhdh-branding-logo.adoc:7]: Consider applying procedure template:

* Add the `.Procedure` subtitle.
* Steps are an ordered list.

Comment (hmanwani-rh) [reply modules/customization/proc-customize-rhdh-branding-logo.adoc:7]: @themr0c The suggestion is outside the scope of this PR. I recommend applying the template to the other sections currently part of this chapter as well. If you'd like, I can create a Jira for these suggestions and work on applying the templates through a separate Jira. The scope of this PR is solely to restore the archived content.

**Pull request merged**